### ISSUE TITLE
Bytecode similarity search

### ISSUE THREAD

**Issue by kuzdogan** [ethereum/sourcify#1642]

**Comment by kuzdogan** [comment]
TODO: create issues from the discussion here https://github.com/ethereum/sourcify/issues/1643

**Comment by manuelwedler** [comment]
Should be worth to have a look at the Blockscout bytecode database service for this: https://docs.blockscout.com/devs/verification/ethereum-bytecode-database-microservice

**Comment by marcocastignoli** [comment]
> Should be worth to have a look at the Blockscout bytecode database service for this: https://docs.blockscout.com/devs/verification/ethereum-bytecode-database-microservice

I read through this but it doesn't seem like they are doing anything special for the bytecode search. They just take into account bytecode auxdata from what I understood.

**Comment by kuzdogan** [comment]
Yes I think they are not doing similarity but looking for byte-by-byte matches (except cborAuxdata). 

Of course if we find a byte-by-byte match, that will be the correct response. The similarity search comes if it this fails
